repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/samlpolicy/SamlFlow10Test.groovy

```diff
@@ -128,7 +130,16 @@ class SamlFlow10Test extends ReposeValveTest {
 
     @Unroll
     def "a saml:response that #signedState will not be altered and will maintain signature validity through Repose"() {
-        when:
+        given: "the Identity mock will return a mapping policy with extended attributes should one be requested (it shouldn't be)"
```

Comment:
Yeah, I could set it to `null` instead.  It would make the test a little cleaner (aside from using a `null`).